lua/conform/formatters/prettierd.lua

```diff
@@ -1,5 +1,66 @@
 local fs = require("conform.fs")
 local util = require("conform.util")
+
+local config_file_names = {
+  -- https://prettier.io/docs/en/configuration.html
+  ".prettierrc",
+  ".prettierrc.json",
+  ".prettierrc.yml",
+  ".prettierrc.yaml",
+  ".prettierrc.json5",
+  ".prettierrc.js",
+  ".prettierrc.cjs",
+  ".prettierrc.mjs",
+  ".prettierrc.toml",
+  "prettier.config.js",
+  "prettier.config.cjs",
+  "prettier.config.mjs",
+}
```

Comment:
No issue from me, but I just wanted to point out the fix I proposed for prettier (not prettierd) in #567 - specifically, these formatters should not run on ALL projects, meaning that if package.json is detected, they run. Some projects have other formatters and keeping package.json here will trigger prettier alongside their preferred formatter, leading to linting errors and screen flicker because of the different formatting styles being applied one after another.

Reply:
Yes, treating package.json existence as a sufficient condition for the prettierd being enabled is not correct. Ignoring package.json is not correct as well. The point of the PR is to fix the package.json treatment. And yes, I'd like to share this logic with the prettier formatter if this PR finds it's way to the master.